### PR TITLE
Do not include heartbeat URLs in catchall

### PR DESCRIPTION
**Pull request by rehandalal**
r?

### CODE REVIEW

File: morgoth/base/tests/test_app.py [hunk 1]
```diff
@@ -1,4 +1,27 @@
...
+        assert res.status_code == 200
+
+        res = client.get('/__lbheartbeat__/')
+        assert res.status_code == 404
+
+        res = client.get('/__heartbeat__/')
+        assert res.status_code == 404
+
+        res = client.get('/__version__/')
+        assert res.status_code == 404
```
Comment (mythmon) [inline morgoth/base/tests/test_app.py]: I don't understand why this view would 404. Is it the trailing slash?

Comment (rehandalal) [inline morgoth/base/tests/test_app.py]: i'm using a negative lookahead in the routing regex: `(?!(__lbheartbeat__)|(__heartbeat__)|(__version__))`

Comment (rehandalal) [inline morgoth/base/tests/test_app.py]: so it doesn't send those particular URLs to the `index` view

Comment (mythmon) [inline morgoth/base/tests/test_app.py]: Oh, I missed that part. Maybe a comment in the url routes would be good?

Also, how will this act when you actually have heartbeat and version endpoints?

**Comment by mythmon** [comment]
Looks good to me. r+

It might be time to start pulling out that health app into a separate package. It looks like it was nicely portable here.

**Pull request merged**